### PR TITLE
feat(cli): add fanout plan subcommand

### LINKED ISSUE #215: feat(cli): fanout plan サブコマンド — spec 実行・dry-run・冪等性・Tier2 ゴールデン
## Goal

新サブコマンド `fanout plan <spec.json | plan-slug>` を実装する: spec を読み込み、worktree + tmux ペイン + ブリーフィング + state 記録 + エージェント起動を決定論的に実行する。フラグサーフェスと挙動の単一の真実は親 issue #211 の「CLI サーフェス」。

## Scope

- `cmd/fanout/main.go`: `update`/`dashboard` と同様の early-return dispatch を 1 箇所追加(`args[0] == "plan"`)。
- `cmd/fanout/plancmd.go`(**新規**。`cmd/fanout/plan.go` は issue モードの既存ファイルなので触らない):
  - 専用パーサ(`--agent` / `--dry-run` / `--limit` / `--only`・`--skip`(task-id CSV、排他)/ `--unblocked-only` / `--base-branch` / `--branch-prefix` / `--no-refresh` / `--session` / `--sleep` / `--debug` / settings 系 on/off スイッチ)。`--include`・`--name`・`--codex-plan-mode` は受け付けない。
  - 依存チェック: 起動パスは git + tmux のみ。gh は `--unblocked-only` のときだけ要求。gh-sub-issue は不要。
  - 実行フロー: resolveRuntime → settings.Resolve → planspec.Load/Validate → state ロック取得(CLAUDE.md のロック近接ルール: フィルタ・起動の直前)→ `FannedTaskIDsForParent` + worktree 存在フォールバック(exact-name match)→ フィルタ(--only/--skip → 登録済み → blocked → --limit)→ fail-fast ループで createPane → サマリ + 繰り越し分の rerun ヒント。
  - `--unblocked-only`: spec の blocked_by グラフ + 完了状態(state 行消滅 かつ worktree 不在、または `ghissue.PRsForBranch` による head ブランチの MERGED PR)で判定。
  - live 実行時に spec を `.fanout/plans/<slug>.json` へ atomicfs でコピー。live 成功後は `bindDashboardKey` を呼ぶ(issue モードと同等)。
- `cmd/fanout/pane.go`: `paneRequest.TaskID` 追加、タスク版 request builder(`briefing.RenderTask`/`TaskPath` を使用)とタスク版 one-line prompt(`[fanout <task-id> of plan:<slug>] ...`)。`statePane` が TaskID をコピーし、rollback は `RemoveTaskPane` を使う。
- `internal/worktree/worktree.go`: ローカル exclude パターンに `.fanout/plans/` を追加。
- テスト:
  - Tier1: spec 不在 / 不正 JSON / version 不一致 / id 重複 / 循環 / `--only`+`--skip` 排他 / 未知フラグ → 既存規約どおりの exit code とメッセージ。
  - Tier2: fixtures `scenario-plan-basic` / `scenario-plan-idempotency`(taskId 行を seed した state.json)/ `scenario-plan-blocked`(gh shim の `pr list --head` 分岐)/ `scenario-plan-limit` + goldens、`tests/bats/helpers.bash` にヘルパー追加、`tests/bin/gh` に `pr list --head` 分岐追加。

## Acceptance criteria

- [ ] `scenario-plan-basic` の dry-run が gh 呼び出しゼロで golden 一致
- [ ] seed 済み state での再実行が 0 ペイン作成(冪等)
- [ ] `--unblocked-only` が head ブランチの MERGED PR でブロッカー解除を判定
- [ ] 最初の失敗で停止する fail-fast が維持され、rollback で task 行が消える
- [ ] 既存の issue モード Tier2 golden が無変更で全パス
- [ ] `make test` / `make lint` クリーン

## Notes

- デザイン/フロントエンド: 不要。
- 依存先が提供する部品: planspec(#212)、state.TaskID + PRsForBranch(#213)、RenderTask/TaskPath(#214)。

## Blocked by

- #212
- #213
- #214

### PR DESCRIPTION
**Pull request by butaosuinu**
**TL;DR**
Adds `fanout plan <spec.json|plan-slug>` for issue-less plan task fanout, including dry-run output, idempotent task rows, dependency gating, and dashboard/TUI task-row rendering. Review effort: 4

**Why**
Issue #215 asks for a CLI path that can execute a structured plan spec without GitHub sub-issues while preserving the existing fanout guarantees around dry-runs, idempotency, and Tier2 golden coverage.

```mermaid
flowchart TD
  main[cmd/fanout/main.go] --> plan[cmd/fanout/plancmd.go]
  plan --> spec[internal/planspec]
  plan --> pane[cmd/fanout/pane.go]
  pane --> state[internal/state]
  state --> session[internal/sessionview]
  session --> tui[internal/tui]
  session --> web[web/src]
  plan --> bats[tests/bats]
```

**Changes by file**
<details><summary>Changed files</summary>

| File | What changed | Why |
| --- | --- | --- |
| `README.md` | Documented plan spec execution, dry-run, idempotency, and dependency behavior. | Expose the new user-facing CLI surface. |
| `README.ja.md` | Added Japanese documentation for the plan spec workflow. | Keep the Japanese README aligned with CLI behavior. |
| `cmd/fanout/main.go` | Routed `fanout plan` before legacy issue parsing. | Let plan mode use task IDs instead of issue numbers. |
| `cmd/fanout/pane.go` | Added task pane requests, task prompts, state rows, rollback, and labels. | Reuse pane creation while recording issue-less plan tasks safely. |
| `cmd/fanout/pane_test.go` | Added task pane, prompt, slug, branch, and state tests. | Cover task request identity and prompt behavior. |
| `cmd/fanout/plancmd.go` | Added the `fanout plan` command parser and executor. | Implement spec loading, filtering, dependency gating, dry-run output, state locking, and launches. |
| `cmd/fanout/plancmd_test.go` | Added unit tests for slug/branch validation, blocker gating, and completion evidence. | Lock down idempotency and dependency edge cases. |
| `internal/cliflags/usage.go` | Added `fanout plan` to top-level usage. | Make the subcommand discoverable. |
| `internal/dashboard/poller.go` | Skipped GitHub wave/PR fetches for task-only rows and pruned plan-safe cache state. | Avoid treating `issueNum=0` task rows as GitHub issues. |
| `internal/dashboard/poller_test.go` | Adjusted dashboard poller expectations for task rows. | Verify task rows do not drive issue fetches. |
| `internal/sessionview/aggregate.go` | Carried task IDs into snapshots and guarded GitHub fetches to positive issue numbers. | Render task rows without issue collisions. |
| `internal/sessionview/aggregate_test.go` | Added task-row snapshot coverage. | Verify task rows are included and isolated from synthetic issue rows. |
| `internal/sessionview/model.go` | Added `TaskID` to pane view data. | Carry task identity to TUI/web consumers. |
| `internal/tui/tui.go` | Displayed/search task IDs and blocked close/merge/cleanup for plan task rows. | Prevent issue lifecycle commands from acting on `issueNum=0`. |
| `internal/tui/tui_test.go` | Added task-row display/filter/lifecycle tests. | Cover the new TUI behavior. |
| `internal/worktree/worktree.go` | Ignored `.fanout/plans/` in local git exclude. | Keep copied live plan specs out of git status. |
| `tests/bats/helpers.bash` | Added helpers for plan fixture execution. | Support deterministic plan Tier2 scenarios. |
| `tests/bats/tier1_flags.bats` | Added plan flag/parser coverage. | Validate task-id filters and unsupported issue-mode flags. |
| `tests/bats/tier2_dry_run.bats` | Added plan dry-run golden scenarios. | Cover dry-run output, idempotency, dependencies, completion, and limits. |
| `tests/bin/gh` | Added `gh pr list --head` fixture support. | Let plan completion checks run in Tier2 without live GitHub. |
| `tests/fixtures/scenario-plan-basic/plan.json` | Added a basic two-task plan spec. | Fixture for basic plan dry-run. |
| `tests/fixtures/scenario-plan-basic/project_root/.gitkeep` | Added fixture project root marker. | Keep the fixture root in git. |
| `tests/fixtures/scenario-plan-basic/tmux-sessions.txt` | Added tmux session fixture. | Support dry-run runtime resolution. |
| `tests/fixtures/scenario-plan-blocked/gh-pr-list-head-fanout__launch-plan-define-base-types-base-types.json` | Added merged blocker PR fixture. | Model a completed dependency. |
| `tests/fixtures/scenario-plan-blocked/gh-pr-list-head-fanout__launch-plan-extract-api-client-api-client.json` | Added empty dependent PR fixture. | Model an incomplete dependent task. |
| `tests/fixtures/scenario-plan-blocked/plan.json` | Added dependency plan spec. | Fixture for unblocked-only gating. |
| `tests/fixtures/scenario-plan-blocked/project_root/.fanout/state.json` | Added seeded task state. | Model an already fanned blocker task. |
| `tests/fixtures/scenario-plan-blocked/project_root/.fanout/worktrees/launch-plan-define-base-types-base-types/.gitkeep` | Added seeded worktree marker. | Preserve fixture worktree path. |
| `tests/fixtures/scenario-plan-blocked/project_root/.gitkeep` | Added fixture project root marker. | Keep the fixture root in git. |
| `tests/fixtures/scenario-plan-blocked/tmux-sessions.txt` | Added tmux session fixture. | Support dry-run runtime resolution. |
| `tests/fixtures/scenario-plan-complete-blocker/gh-pr-list-head-fanout__launch-plan-define-base-types-base-types.json` | Added merged blocker PR fixture. | Model a completed selected dependency. |
| `tests/fixtures/scenario-plan-complete-blocker/gh-pr-list-head-fanout__launch-plan-extract-api-client-api-client.json` | Added empty dependent PR fixture. | Keep dependent completion evidence absent. |
| `tests/fixtures/scenario-plan-complete-blocker/plan.json` | Added completed-blocker plan spec. | Fixture for skipping completed selected blockers. |
| `tests/fixtures/scenario-plan-complete-blocker/project_root/.gitkeep` | Added fixture project root marker. | Keep the fixture root in git. |
| `tests/fixtures/scenario-plan-complete-blocker/tmux-sessions.txt` | Added tmux session fixture. | Support dry-run runtime resolution. |
| `tests/fixtures/scenario-plan-complete-leaf/gh-pr-list-head-fanout__launch-plan-build-ui-shell-ui-shell.json` | Added merged leaf PR fixture. | Model a completed terminal task. |
| `tests/fixtures/scenario-plan-complete-leaf/plan.json` | Added completed leaf plan spec. | Fixture for skipping completed terminal tasks. |
| `tests/fixtures/scenario-plan-complete-leaf/project_root/.gitkeep` | Added fixture project root marker. | Keep the fixture root in git. |
| `tests/fixtures/scenario-plan-complete-leaf/tmux-sessions.txt` | Added tmux session fixture. | Support dry-run runtime resolution. |
| `tests/fixtures/scenario-plan-fresh-blocker/gh-pr-list-head-fanout__launch-plan-define-base-types-base-types.json` | Added empty blocker PR fixture. | Model missing completion evidence. |
| `tests/fixtures/scenario-plan-fresh-blocker/gh-pr-list-head-fanout__launch-plan-extract-api-client-api-client.json` | Added empty dependent PR fixture. | Keep dependent completion evidence absent. |
| `tests/fixtures/scenario-plan-fresh-blocker/plan.json` | Added fresh blocker plan spec. | Fixture for incomplete dependency blocking. |
| `tests/fixtures/scenario-plan-fresh-blocker/project_root/.gitkeep` | Added fixture project root marker. | Keep the fixture root in git. |
| `tests/fixtures/scenario-plan-fresh-blocker/tmux-sessions.txt` | Added tmux session fixture. | Support dry-run runtime resolution. |
| `tests/fixtures/scenario-plan-idempotency/plan.json` | Added idempotency plan spec. | Fixture for seeded task rows. |
| `tests/fixtures/scenario-plan-idempotency/project_root/.fanout/state.json` | Added seeded task state rows. | Verify task ID idempotency. |
| `tests/fixtures/scenario-plan-idempotency/project_root/.gitkeep` | Added fixture project root marker. | Keep the fixture root in git. |
| `tests/fixtures/scenario-plan-idempotency/tmux-sessions.txt` | Added tmux session fixture. | Support dry-run runtime resolution. |
| `tests/fixtures/scenario-plan-limit/plan.json` | Added three-task plan spec. | Fixture for `--limit` behavior. |
| `tests/fixtures/scenario-plan-limit/project_root/.gitkeep` | Added fixture project root marker. | Keep the fixture root in git. |
| `tests/fixtures/scenario-plan-limit/tmux-sessions.txt` | Added tmux session fixture. | Support dry-run runtime resolution. |
| `tests/golden/scenario-plan-basic.dry-run.txt` | Added basic plan dry-run golden. | Assert deterministic task pane output. |
| `tests/golden/scenario-plan-blocked.dry-run.txt` | Added dependency-unblocked golden. | Assert completed blockers unlock dependents. |
| `tests/golden/scenario-plan-complete-blocker.dry-run.txt` | Added completed blocker golden. | Assert selected completed blockers are skipped. |
| `tests/golden/scenario-plan-complete-leaf.dry-run.txt` | Added completed leaf golden. | Assert completed terminal tasks are skipped. |
| `tests/golden/scenario-plan-fresh-blocker.dry-run.txt` | Added fresh blocker golden. | Assert missing completion evidence blocks dependents. |
| `tests/golden/scenario-plan-idempotency.dry-run.txt` | Added idempotency golden. | Assert seeded task rows are skipped. |
| `tests/golden/scenario-plan-limit.dry-run.txt` | Added limit golden. | Assert plan rerun hints for deferred task IDs. |
| `web/src/components/Drawer.tsx` | Displayed task labels in drawer details. | Make issue-less task rows readable. |
| `web/src/components/SessionSection.tsx` | Used task-safe row keys and labels. | Avoid key collisions with `issueNum=0`. |
| `web/src/lib/filter.test.ts` | Added task filter coverage. | Verify task IDs are searchable. |
| `web/src/lib/filter.ts` | Added `task:` structured filter and task ID free text. | Support filtering plan task rows. |
| `web/src/lib/pane.test.ts` | Added pane identity helper tests. | Verify issue and task row keys/labels. |
| `web/src/lib/pane.ts` | Added task-aware row keys and labels. | Avoid collisions and show task IDs. |
| `web/src/lib/types.ts` | Added `taskId` to frontend pane type. | Match the snapshot schema. |

</details>

**Risk**
The main behavior risk is interaction between task rows (`issueNum=0`) and existing issue-only lifecycle/status paths. This PR keeps lifecycle actions issue-only, skips GitHub issue lookups for task rows, and covers those paths with Go, web, and Tier2 tests.

**Test plan**
- `make test` - passed.
- `make lint` - passed.
- `make lint-web` - passed via `codex review --uncommitted`.
- `git diff --check` - passed.
- `codex review --uncommitted` - passed with no findings.
- `./fanout-go msg inbox --self 215 --parent 211` - passed; only board notes from #213 and #216 were present.
- `./fanout-go msg board --self 215 --parent 211` - passed; same board notes from #213 and #216 were present.

Closes #215

### CODE REVIEW

File: cmd/fanout/plancmd.go [hunk 1]
```diff
@@ -0,0 +1,813 @@
...
+		return
+	}
+	fmt.Fprintf(lg.Stdout(), "\n%sDeferred %d task(s) due to --limit. Rerun with:%s\n", c.Info, len(plan.LimitDeferred), c.Reset)
+	ids := taskIDCSV(plan.LimitDeferred)
+	fmt.Fprintf(lg.Stdout(), "  %s\n", strings.Join(taskIDs(plan.LimitDeferred), " "))
+	cliCfg := cfg.cliConfig()
+	fmt.Fprintf(lg.Stdout(), "  %s plan %s --only %s%s%s%s%s%s%s\n",
+		shellQuote(commandName),
+		shellQuote(cfg.SpecArg),
+		shellQuote(ids),
```
[the remark this answers is not in the document]
Comment (butaosuinu) [reply cmd/fanout/plancmd.go:797]: 対応しました。live run では spec copy 後の plan slug を --limit rerun hint に使い、dry-run では元の spec 引数を維持するようにしました。commit: feaf564

File: cmd/fanout/plancmd.go [hunk 2]
```diff
@@ -0,0 +1,813 @@
...
+
+	cfg.SpecPath = resolvePlanSpecPath(rt.info.ProjectRoot, cfg.SpecArg)
+	spec, err := planspec.Load(cfg.SpecPath)
+	if err != nil {
+		lg.Err("%v", err)
+		return exitcode.Env
+	}
+	if cfg.BaseBranch == "" {
+		cfg.BaseBranch = spec.Plan.BaseBranch
+	}
```
[the remark this answers is not in the document]
Comment (butaosuinu) [reply cmd/fanout/plancmd.go]: 対応しました。spec 由来の plan.base_branch を CLI と同じ whitespace validation に通し、CLI --base-branch が指定されている場合だけ spec 値を無視するようにしました。commit: feaf564

File: cmd/fanout/plancmd.go [hunk 3]
```diff
@@ -0,0 +1,813 @@
...
+		AutoPullRequest:    cfg.AutoPullRequest,
+		PRReviewGate:       cfg.PRReviewGate,
+		BriefingCodeReview: cfg.BriefingCodeReview,
+		AgentTeamsHint:     cfg.AgentTeamsHint,
+		PRVisualization:    cfg.PRVisualization,
+		DashboardKeybind:   cfg.DashboardKeybind,
+	}, lg.Warn)
+
+	cfg.SpecPath = resolvePlanSpecPath(rt.info.ProjectRoot, cfg.SpecArg)
+	spec, err := planspec.Load(cfg.SpecPath)
```
[the remark this answers is not in the document]
Comment (butaosuinu) [reply cmd/fanout/plancmd.go]: 対応しました。fanout plan は planspec の schema/dependency validation を通しつつ、中間 resolved slug/branch の duplicate check は skip し、CLI の plan-qualified slug / branch-prefix 適用後に final name collision を検証するようにしました。commit: feaf564
